lib/puppet/provider/service/systemd.rb

```diff
@@ -114,15 +114,6 @@ def debian_enabled?(svc_info)
     end
   end
 
-  def status
-    begin
-      systemctl("is-active", @resource[:name])
-    rescue Puppet::ExecutionFailure
```

Comment:
This is now relying on `texecute` from base service provider in `service.rb` to catch `Puppet::ExecutionFailure`, which calls `fail`.  I suspect this is done this way on purpose to report `:stopped` when `is-active` returns a non-zero code, perhaps?